src/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from "react";
+
+import { BaseBadgeProps } from "./type";
+import { StyledBadge } from "./Style";
+import { ReactComponent as IconIcBlack } from "../../assets/icon-ic-black.svg";
+import { ReactComponent as IconIcBlue } from "../../assets/icon-ic-blue.svg";
+import { ReactComponent as IconIcGreen } from "../../assets/icon-ic-green.svg";
+import { ReactComponent as IconIcOrange } from "../../assets/icon-ic-orange.svg";
+import { ReactComponent as IconIcRed } from "../../assets/icon-ic-red.svg";
+import { ReactComponent as IconIcWhite } from "../../assets/icon-ic-white.svg";
+
+const Badge: FC<BaseBadgeProps> = (props) => {
+  const {
+    variant = "light",
+    inverted = false,
+    suffix = false,
+    prefix,
+    iconOnly = false,
+    label = "",
+    children,
+  } = props;
+
+  return (
+    <StyledBadge variant={variant} inverted={inverted}>
+      <div className="badgeWrapper">
+        {!iconOnly ? (
+          <>
+            <div className="prefixBadge">{prefix}</div>
+            <p className="badgeLabel">{label}</p>
+            <>{suffix && renderSuffixIconBadge({ inverted, variant })}</>
+          </>
+        ) : (
+          children
```

Comment:
buat `label` dan `children` ini kan dipisah? menurut u apakah enaknya dipisah gini ato dijadiin satu aja children gt instead of label dan children?

Reply:
hmm iya, sebenernya children itu buat nampung icon ketika iconOnly == true sih mar. Sementara label buat nampilin label badge semisal ada. Mungkin gue ganti penamaannya aja ya biar ga membingungkan

Reply:
mm kalo misalnya label diilangin, kalo iconOnly == false, render children dan suffix, terus suffix bakal di ignore kalo iconOnly == true dan ngerender children, wdyt of that idea?

Reply:
bisa sih, berarti type children tetep dibikin ReactNode instead of text ya

Reply:
yup

Reply:
reactnode itu jg bisa text kok

Reply:
done ya